### PR TITLE
Editing branch

### PR DESCRIPTION
**Pull request by Petros17**
Added a folder named gandalf to english section

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2353)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
Sorry about the delay, @Petros17! @casey-collab, please review this.

File: english/gandalf/gandalf.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Suddenly, you hear a music behind you and feel a presence.
```
Comment (casey-collab) [inline english/gandalf/gandalf.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**